packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -60,15 +61,25 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
     async function getEstimatedGas() {
       if (arbTokenBridge.token && token?.address) {
```

Comment:
I think it's now safe to remove `arbTokenBridge.token` from the check and the dependency array

Reply:
Makes sense. Done.